android/app/src/main/java/io/mosip/registration_client/MainActivity.java

```diff
@@ -138,11 +144,24 @@ public void configureFlutterEngine(@NonNull FlutterEngine flutterEngine) {
     }
 
     public void getUISchema(MethodChannel.Result result){
+        try{
+
+            String schemaJson = identitySchemaRepository.getSchemaJson(getApplicationContext(),
+                    identitySchemaRepository.getLatestSchemaVersion());
+            result.success(schemaJson.toString());
+
+        }catch (Exception e){
+
+        }
+    }
+    public void getNewProcessSpec(MethodChannel.Result result){
         try{
 
             ProcessSpecDto processSpecDto = identitySchemaRepository.getNewProcessSpec(getApplicationContext(),
                     identitySchemaRepository.getLatestSchemaVersion());
-            result.success(processSpecDto.toString());
+            ObjectWriter ow = new ObjectMapper().writer().withDefaultPrettyPrinter();
```

Comment:
Instead of creating ObjectMapper everytime, can we create one objectMapper instance and reuse.

Reply:
Done